backend/pom.xml

```diff
@@ -1,97 +1,118 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Looks like the entire file was reformatted. Did that happen automatically with one of the auto-formaters? Were there any other changes to this file or just addition / removal of blank space?

Reply:
I might've used intellij's auto-format when i had reverted the file which caused the spacing changes but in terms of additions i did add  additional dependencies which were just the dependencies needed for Gmail such as oath client etc.